<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateBillDialog</class>
 <widget class="QDialog" name="CreateBillDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BillSplittr</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(236, 236, 236);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="billSplittr">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
font: bold;
color: #3282b8;
border: none</string>
       </property>
       <property name="text">
        <string>BillSplittr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTime">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>BlankCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
font: bold;
color: #3282b8;
border: none;</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="keyboardTracking">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="numPeople_label">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
font: bold;
color: #0f4c75;</string>
       </property>
       <property name="text">
        <string>Number of People:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numPeople">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16px;
font: bold;
color: #0f4c75;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="total_label">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
font: bold;
color: #0f4c75;</string>
       </property>
       <property name="text">
        <string>Total:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="total">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16px;
font: bold;
color: #0f4c75;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #3282b8;
border:outset;
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="payer_label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16px;
font: bold;
color: #0f4c75;</string>
     </property>
     <property name="text">
      <string>웃 Payer</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="name_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
font: bold;
color: #3282b8;</string>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pay_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
font: bold;
color: #3282b8;</string>
       </property>
       <property name="text">
        <string>Pay  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="billName_scrollArea">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>230</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #dae1e7;
border-radius: 10px;
font: 14px;
font-weight: bold;
color: #0f4c75;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="billName_scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>376</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="list_label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16px;
font: bold;
color: #0f4c75;</string>
     </property>
     <property name="text">
      <string>☰ List</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="listName_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
font: bold;
color: #3282b8;</string>
       </property>
       <property name="text">
        <string>List Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="price_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
font: bold;
color: #3282b8;</string>
       </property>
       <property name="text">
        <string>Price</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="perPerson_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14px;
font: bold;
color: #3282b8;</string>
       </property>
       <property name="text">
        <string>Per Person</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="listName_scrollArea">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>230</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #dae1e7;
border-radius: 10px;
font: 14px;
font-weight: bold;
color: #0f4c75;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="billList_crollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>376</width>
        <height>230</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
